src/pages/CheckoutDetails.jsx

```
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)